test/distributions/replicated/testReplicand.good

```diff
@@ -0,0 +1,4 @@
+0 0 0 0 0
+1 1 1 1 1
+2 2 2 2 2
+3 3 3 3 3
```

Comment:
Seems like you either need a testReplicand.comm-none.good or a skipif for `CHPL_COMM==none`?  (I tend to prefer the former to make sure that things work on single-locale runs as expected).